src/queries/ScheduledEventQueries.ts

```diff
@@ -34,15 +35,17 @@ export default class ScheduledEventQueries {
     return this.scheduledEventDataSource.getAll(filter);
   }
 
-  @Authorized([Roles.USER_OFFICER, Roles.INSTRUMENT_SCIENTIST])
+  // @Authorized([Roles.USER_OFFICER, Roles.INSTRUMENT_SCIENTIST])
```

Comment:
I'm planning to fix this in this PR, but in the meantime, it should be ready for review.